New package: EDCore v0.1.2

JuliaRegistrator opened the pull request:
- Registering package: EDCore
- Repository: https://github.com/Zou-Bo/MomentumED.jl
- Created by: @Zou-Bo
- Version: v0.1.2
- Commit: 9d639fe656663987e3f11d2deae4cf5d9931d9c7
- Reviewed by: @Zou-Bo
- Reference: https://github.com/Zou-Bo/MomentumED.jl/commit/9d639fe656663987e3f11d2deae4cf5d9931d9c7#commitcomment-169475515
- Description: A Julia package that does Exact Diagonalization calculation for 2D systems in momentum basis. Momentum conserved Hamiltonian is required.
<!-- 97e20d8018a1bbf82b03253b805eeadcfc7538b92da8fc1e46143e724aa432669cfd7da63cbf360b06ad9e1b05a729af94fda8631474246f468c6b8c40f75a6c66ee4298d5c34656c16084c20d0a891a4be1efc1d750713bb57694399c639c90445cfb4443e177cbf1d42b8b93bfd003f8fc85bac251beeacd634b1fa6c26ad55784145de261c0f7f2454838f313287286af099a7f2c1720529e4c57e9e6a3e0f9fe3aa784b97da90f50804c5c9824cf8f24dda8a69a29028549a193c743f2b2856b197ed8624c72e10bd113efc98efb59a4ebe1c2300e3e3162ce625c3d5e9d -->

goerz commented:
[noblock] The name similarity at least is a false positive, so I’ve applied the override for that

JuliaRegistrator committed:
New package: EDCore v0.1.2

UUID: 67ba95b5-0c7c-4a9c-9737-2f4756a96b17
Repo: https://github.com/Zou-Bo/MomentumED.jl.git
Tree: 4067c8b3243a382dce1c6206bb4a3fe2c0bc7e4b

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
